### ISSUE TITLE
merge conflict on hotfix-finish to develop branch

### ISSUE THREAD

**Issue by Linkje** [aleksandr-m/gitflow-maven-plugin#401]
Hi,

We have noticed that when you use the the hotfix-finish command we get a merge conflict on all pom files 

[ERROR] Failed to execute goal com.amashchenko.maven.plugin:gitflow-maven-plugin:1.20.0:hotfix-finish (default-cli) on project reactor: hotfix-finish: 
[ERROR] Auto-merging parent/pom.xml
[ERROR] CONFLICT (content): Merge conflict in parent/pom.xml

We saw that the outputTimestamp was different in develop vs the hotfix branch but even when we disable updating the outputTimestamp we see the same behaviour.

Skipping the backmerge to develop and creating a seperate PR to develop fixes our workflow for now. Any suggestions what would cause this?

**Comment by sfcoy** [comment]
We are seeing the same behaviour. Here is our plugin config:

            <plugin>
                <groupId>com.amashchenko.maven.plugin</groupId>
                <artifactId>gitflow-maven-plugin</artifactId>
                <inherited>false</inherited>
                <configuration>
                    <gitFlowConfig>
                        <productionBranch>main</productionBranch>
                        <developmentBranch>develop</developmentBranch>
                    </gitFlowConfig>
                    <versionProperty>revision</versionProperty>
                    <skipUpdateVersion>true</skipUpdateVersion>
                    <useSnapshotInHotfix>true</useSnapshotInHotfix>
                    <useSnapshotInRelease>true</useSnapshotInRelease>
                    <useSnapshotInSupport>true</useSnapshotInSupport>
                </configuration>
            </plugin>

**Comment by Linkje** [comment]
Yes is still happening to us as well. Another workaround has been to merge the outputtimestamp of develop into the hotfix branch which resolves the conflict.
